lib/select_dialog.dart

```diff
@@ -42,6 +44,7 @@ class SelectDialog<T> extends StatefulWidget {
     this.onFind,
     this.itemBuilder,
     this.searchBoxDecoration,
+    this.searchHint,
```

Comment:
Maybe should move the default value to the constructor to make it more explicit.
```dart
this.searchHint = "Find",
```

Reply:
Actually tried this but the default case isn't working for me if I use `showModal` constructor.
The default case should be 'Find' but the line 
`searchHint: searchHint,`
seams to be considered an assignment to the parameter also if `searchHint` is `null`.
The result is that instead of getting the default value 'Find' I get a `null` string and no hint in the search box.